tasks.py

```diff
@@ -123,12 +126,21 @@ def parse_spec_for_filters(_):
     )
 
     filters = []
+    optional_filters = []
     for line in specification_flines:
         if ":filter:" in line:
+            print(line)
             for _split in line.replace("filter=", "").split(":filter:")[1:]:
                 _filter = _split.split("`")[1].strip()
-                filters.append(_filter)
+                if "OPTIONAL" in line:
```

Comment:
Suggestion: Make `"OPTIONAL"` a variable so it's more easily changed, if this should ever become an issue.

Reply:
Good idea, I've made this a list so we can append if necessary